discord/ext/commands/view.py

```diff
@@ -87,20 +87,20 @@ def skip_ws(self):
 
     def skip_string(self, string):
         strlen = len(string)
-        if self.buffer[self.index : self.index + strlen] == string:
+        if self.buffer[self.index: self.index + strlen] == string:
```

Comment:
Shouldn't these spaces be there according to pep8

Reply:
It's not entirely clear as `pycodestyle` raises a weak warning with that but it does appear pep8 wants it that way. Will revert these.